tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"version": "2.0.3",
+	"compilerOptions": {
+		"declaration": false,
+		"module": "umd",
+		"noImplicitAny": true,
+		"noImplicitThis": true,
```

Comment:
I'm not sure. I think the last change was to add `strictNullChecks`: https://github.com/dojo/widgets/blob/master/tsconfig.json#L4-L15.  Although if there are more that I **should** add I can of course put those in.